src/app/player-creation/services/avatar-generator.service.ts

```diff
@@ -1,5 +1,20 @@
 import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
 
 @Injectable()
 export class AvatarGenerator {
+  private readonly url = 'https://proxy.cors.sh/https://api.multiavatar.com/';
```

Comment:
I think we should add some kind of default avatar here still. As we know from random name generator, this CORS api is rate limited and the call will fail after a time :( 
In general, maybe we should set up our own proxy to add the Origin headers or someday. If we need it for the avatars we depend to much on it working.

Reply:
or maybe we just look for another API. not in this PR though :)